gslib/addlhelp/creds.py

```diff
@@ -40,23 +40,24 @@
   credentials using the gcloud auth command as well (for more details, see
   https://cloud.google.com/sdk/gcloud/reference/auth).
 
-  Once credentials have been configured via gcloud auth, those credentials will
-  be used regardless of whether the user has any boto configuration files (which
-  are located at ~/.boto unless a different path is specified in the BOTO_CONFIG
-  environment variable). However, gsutil will still look for credentials in the
-  boto config file if a type of non-GCS credential is needed that's not stored
-  in the gcloud credential store (e.g., an HMAC credential for an S3 account).
+  Once credentials have been configured via ``gcloud auth``, those credentials
+  are used regardless of whether the user has any boto configuration files
+  (which are located at ~/.boto unless a different path is specified in the
+  BOTO_CONFIG environment variable). However, gsutil still looks for credentials
+  in the boto config file if a type of non-GCS credential is needed that's not
+  stored in the gcloud credential store (e.g., an HMAC credential for an S3
```

Comment:
This might be impacted by the fact that we're adding HMAC support to gcloud. Not sure if we plan on adding it to the wrapper script and passing HMAC from gcloud to gsutil @thomasmaclean

Reply:
If it is impacted, would it be possible to update the wording here separately? I only made superficial changes to this paragraph - it will continue to say basically the same thing it has been saying up to now.

Reply:
@thomasmaclean should probably approve b/c he did the HMAC design

Reply:
Although you will be able to specify the HMAC keys through properties in gcloud, the shim will have the ability pull in HMAC keys from the boto file, so this logic still stands.